deploy/web/create.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+RESOURCE_GROUP=$1
+APP_NAME=$2
+APP_SVC_PLAN=$3
+IMAGE_NAME=$4
+IMAGE_VERSION=$5
+#DOCKER_USER=$6
```

Comment:
Is there a reason to keep the commented out variables if they are not used?

Reply:
no, i think now i have proven the 'lack of support' for it via the CLI, we will solely use the ACR.  will remove